tools/odbc/CMakeLists.txt

```diff
@@ -5,6 +5,17 @@ if(NOT ODBC_FOUND)
   message(FATAL_ERROR No ODBC found)
 endif()
 
+set(LINK_LIB_LIST "")
```

Comment:
Perhaps rename this to something more descriptive, e.g. `ODBCINST_LINK_LIB`?

Reply:
I thought in reusing this list for other client libs, case needed.